src/files.ts

```diff
@@ -39,7 +38,7 @@ export function notEmpty<T>(value: T | null | undefined): value is T {
   return value !== null && value !== undefined;
 }
 
-const microMatchOptions = { basename: true, dot: true, posixSlashes: true };
+const miniMatchOptions = { matchBase: true, dot: true };
```

Comment:
optional: seems like this should be `multiMatchOptions`

Reply:
multimatch uses `minimatch` underneath and uses minimatch options structure. We could rename it but I don't think it's needed.